modules/v4l2/v4l2.c

```diff
@@ -411,6 +411,25 @@ static int read_frame(struct vidsrc_st *st)
 }
 
 
+static void set_available_devices(struct vidsrc* vs)
+{
+	int i, fd;
+	char name[16];
+
+	for (i=0;i < 16;i++) {
+
+		sprintf(name, "/dev/video%i", i);
```

Comment:
better to use `snprintf` or `re_snprintf` instead so the buffer size can be specified.